etc/config/erlang.defaults.properties

```diff
@@ -1,7 +1,7 @@
 # Default settings for Erlang
 compilers=&erlang
 compilerType=erlang
-versionFlag=
+versionFlag=-noshell -eval io:fwrite("~s~n", [erlang:system_info(otp_release)]), halt().
```

Comment:
Oh blimey! You had to learn `erlang` for this!? :D

Reply:
Hahaha no! I copied the flags over from the original compiler class